content/source/docs/index.html.markdown

```diff
@@ -10,70 +10,35 @@ show_notification: false
 
-Documentation for Terraform's command-line workflows, including docs for [the `terraform` binary and its subcommands](/docs/cli/commands/index.html).
+### Manage Infrastructure
 
-### [Learn Terraform ➜](https://learn.hashicorp.com/terraform?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) (external site)
+- Use the [Terraform Configuration Language](/docs/language/index.html) to describe infrastructure.
+- Use the [Terraform CLI](/docs/cli/index.html) with the Terraform binary for command-line workflows.
 
--> New users can start here.
 
-Interactive tutorials to teach you how to use Terraform's features. Begin with the [Get Started collection](https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS), then continue with task-specific tutorials or go directly to the [Terraform CLI docs](/docs/cli/index.html).
 
 </div>
 
 <div class="col-md-6 col-sm-12">
 
-### [Terraform Cloud ➜](/docs/cloud/index.html)
-
-Documentation for Terraform Cloud.
-
-Terraform Cloud is an application that helps teams use Terraform together. It manages Terraform runs in a consistent and reliable environment, and includes easy access to shared state and secret data, access controls for approving changes to infrastructure, a private registry for sharing Terraform modules, detailed policy controls for governing the contents of Terraform configurations, and more. Terraform Cloud offers free accounts for small teams, and paid plans with additional feature sets for medium-sized businesses.
-
-### [Terraform Enterprise ➜](/docs/enterprise/index.html)
-
-Documentation for Terraform Enterprise.
-
-Terraform Enterprise is an on-premise distribution of Terraform Cloud. It offers enterprises a private instance of the Terraform Cloud application, with no resource limits and with additional enterprise-grade architectural features like audit logging and SAML single sign-on.
-
-### [Terraform Registry Publishing ➜](/docs/registry/index.html)
-
-Documentation about publishing Terraform providers and modules on the [Terraform Registry](https://registry.terraform.io/).
-
-### [Plugin Development ➜](/docs/extend/index.html)
-
--> If you need to create a new Terraform provider (for a public cloud service or a purely internal service), go here.
+### Collaborate
 
-Documentation about developing Terraform providers, with extensive information about Terraform's internals.
+- [Terraform Cloud](/docs/cloud/index.html) helps teams use Terraform together, enabling version control, data sharing, and more.
+- [Terraform Enterprise](/docs/enterprise/index.html) offers a private instance of Terraform Cloud, with features like access control and governance.
 
-Terraform relies on provider plugins to manage infrastructure resources across a wide variety of infrastructure services. Anyone can make and distribute a Terraform provider for their own service.
+### Develop Plugins
```

Comment:
Don't forget that you can also publish modules, which aren't plugins, on the Terraform Registry. I'm trying to think of how to retitle this section to keep it clean and also accurate, but drawing a bit of a blank... Maybe:

```suggestion
### Develop Modules and Plugins
```